code/game/mecha/equipment/weapons/weapons.dm

```diff
@@ -317,6 +317,19 @@
 	variance = 25
 	harmful = TRUE
 
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/scattershot/syndie //Version used for Dark Gygax
+	equip_cooldown = 2 SECONDS
```

Comment:
This should be at the bottom of all variables.
Name and description always go first.

Reply:
Edited - Basetype has it too so I'm removing it